src/app/core/components/header/header.component.ts

```diff
@@ -1,10 +1,51 @@
 import { Component } from '@angular/core';
 
+export interface IHeaderMenu {
```

Comment:
it should be in models/types/etc folder (i'm not sure how you named it)

Reply:
Done